holoviews/plotting/bokeh/element.py

```diff
@@ -1170,6 +1170,8 @@ def _init_glyph(self, plot, mapping, properties):
             plot_method = plot_method[int(self.invert_axes)]
         if 'legend_field' in properties and 'legend_label' in properties:
             del properties['legend_label']
+        if "name" not in properties:
+            properties["name"] = properties.get("legend_label") or properties.get("legend_field", None)
```

Comment:
I think it defaults to None anyways.
```suggestion
            properties["name"] = properties.get("legend_label") or properties.get("legend_field")
```

Reply:
Correct. It is weird as I'm doing the right thing on the same line... It must have been some refactoring I did :man_shrugging: